docs/develprocess.md

```diff
@@ -858,6 +858,91 @@ genisoimage -o $OUTPUT -v -r -J --joliet-long -V "XCP-ng $VERSION" -c boot/isoli
 isohybrid --uefi $OUTPUT
 ```
 
+## Create unattended ISO file
+
+This guide will provide an example steps on how to create unattended ISO image that can be used for mass deployment.
+
+### Prerequisites
+1. Create an answerfile
+```
+<?xml version="1.0"?>
+    <installation mode="fresh" csrtype="ext">
+        <primary-disk>sda</primary-disk>
+        <keymap>us</keymap>
+        <root-password type="plain">123456</root-password>
+        <hostname>server1.lab</hostname>
+        <bootloader location="partition">grub2</bootloader>
+        <network-backend>vswitch</network-backend>
+        <source type="local"/>
+        <admin-interface name="eth0" proto="static">
+            <ipaddr>192.168.1.10</ipaddr>
+            <subnet>255.255.255.0</subnet>
+            <gateway>192.168.1.1</gateway>
+        </admin-interface>
+        <name-server>192.168.1.1</name-server>
+        <name-server>8.8.8.8</name-server>
+        <ntp-server>192.168.1.1</ntp-server>
+        <ntp-server>192.168.1.2</ntp-server>
+        <timezone>Europe/Sofia</timezone>
+    </installation>
```

Comment:
This is not specific to unattended ISO installs, so should probably be given as an example in the section that is dedicated to answerfiles.

By the way there already is an answerfile example at https://xcp-ng.org/docs/install.html#with-uefi. I'd like to avoid multiple competing examples unless there's a strong reason for it.